.codecov.yml

```diff
@@ -6,7 +6,7 @@ coverage:
   status:
     project:
       default:
-        target: 85%
+        target: 70%
```

Comment:
Please add TODO and link to the issue explaining why it went down and when it needs to be restored.

Reply:
Added